runtime/include/j9cfg.h.in

```diff
@@ -247,6 +247,7 @@ extern "C" {
 #cmakedefine J9VM_OPT_USE_FFI
 #cmakedefine J9VM_OPT_USE_FFI_ONLY
 #cmakedefine J9VM_OPT_VALHALLA_MVT
```

Comment:
`J9VM_OPT_VALHALLA_MVT` doesn't appear to be used anywhere; while you're here, perhaps you should remove this definition and the option in `runtime/cmake/options.cmake`.